PluginBuilder/Controllers/AdminController.cs

```diff
@@ -341,7 +345,11 @@ public async Task<IActionResult> EmailSender(EmailSenderViewModel model)
         {
             return View(model);
         }
-
+        if (!IsValidEmailList(model.To))
+        {
+            TempData[TempDataConstant.WarningMessage] = "Invalid email format in the 'To' field. Please ensure all emails are valid";
+            return RedirectToAction("Plugins");
+        }
```

Comment:
If `To` is invalid on post - we should do `ModelState.AddModelError` and provide notice to user, not redirect to `Plugins`